unidist/core/backends/mpi/core/monitor.py

```diff
@@ -66,6 +66,9 @@ def monitor_loop():
         # Proceed the request
         if operation_type == common.Operation.TASK_DONE:
             task_counter.increment()
+            communication.mpi_isend_object(
```

Comment:
What if we would use sync send here? What would perf be?

Reply:
Tried sync send the performance remains the same.